src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2192,28 +2236,28 @@ void func_808332F4(Player* this, PlayState* play) {
 
 /**
  * Get the appropriate Idle animation based on current `modelAnimType`.
- * This is used as the "primary" idle animation.
+ * This is used as the "normal" idle animation.
```

Comment:
```suggestion
 * This is the default idle animation.
```
I don't think quotes are needed here anymore, and I prefer default over normal.

Reply:
I can change these but do you also think this needs to propagate to for example 
`this->idleType = PLAYER_IDLE_NORMAL;`
I think it makes sense to call these "normal" still

Reply:
I went with default for everything